pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "git-co-evg-base"
-version = "0.5.15"
+version = "0.5.16"
```

Comment:
Can you make this be the `1.0.0? release?

Reply:
Sure, done